CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 3.0.0
+
+- Contains function has been renamed to ContainsPath
```

Comment:
We should mention that `Contains` was deprecated in favor of `ContainsPath`

Reply:
@nathaniel-may Aren't we waiting until API v4 to deprecate that function?

Reply:
No.

Reply:
Deprecate !== remove.